tests/rptest/tests/group_membership_test.py

```diff
@@ -113,3 +118,213 @@ def _check_no_duplicates():
                    10,
                    backoff_sec=2,
                    err_msg="found persistent duplicates in groups listing")
+
+
+class GroupMetricsTest(RedpandaTest):
+    topics = (TopicSpec(name="OnePartition", partition_count=1),
+              TopicSpec(name="MultiplePartitions_1"),
+              TopicSpec(name="MultiplePartitions_2"))
+
+    def __init__(self, ctx, *args, **kwargs):
+
+        extra_rp_conf = dict(default_topic_replications=3, )
+        super(GroupMetricsTest, self).__init__(test_context=ctx,
+                                               num_brokers=3,
+                                               extra_rp_conf=extra_rp_conf)
+        self._ctx = ctx
+
+    def _get_offset_from_metrics(self, group):
+        metric = self.redpanda.metrics_sample("kafka_group_offset")
+        if metric is None:
+            return None
+        metric = metric.label_filter(dict(group=group))
+        res = {}
+        for sample in metric.samples:
+            res[(sample.labels['topic'],
+                 sample.labels['partition'])] = sample.value
+        return res
+
+    @cluster(num_nodes=3)
+    def test_check_value(self):
+        rpk = RpkTool(self.redpanda)
+        topic = next(filter(lambda x: "OnePartition" in x.name,
```

Comment:
here you would use `self.topics[0].name`. if you wanted to give them semantically meaningful identifiers then in the constructor you could do something like `self.one_partition_topic = self.topics[0]` and use that alias.